packages/frontend/core/src/desktop/pages/upgrade-to-team/index.tsx

```diff
@@ -36,6 +39,18 @@ const benefitList: I18nString[] = [
 ];
 
 export const Component = () => {
+  const authService = useService(AuthService);
+  const authStatus = useLiveData(authService.session.status$);
+  const [params] = useSearchParams();
+  const recurring = params.get('recurring');
+
+  if (authStatus === 'unauthenticated') {
+    return <PageNotFound noPermission />;
```

Comment:
should redirect to sign in page

Reply:
Yes, there is a login page here. After logging in, you will return to this upgrade page.